src/main/java/org/intellij/sdk/codesync/Constants.java

```diff
@@ -25,6 +27,9 @@ public static String expanduser(String path) {
     public static String CURRENT_GIT_BRANCH_COMMAND = "git rev-parse --abbrev-ref HEAD";
 
     public static String MAGIC_STRING = "IntellijIdeaRulezzz";
+    public static String DIR_CREATE_EVENT = "VfsEvent[create dir";
```

Comment:
👍

Reply:
I think I have added a TODO to handle this in better way in future, but this was quickest I could fine